src/analysis/special_functions/pow.lean

```diff
@@ -893,6 +893,76 @@ by { convert tendsto_rpow_div_mul_add (1:ℝ) _ (0:ℝ) zero_ne_one, ring_nf }
 lemma tendsto_rpow_neg_div : tendsto (λ x, x ^ (-(1:ℝ) / x)) at_top (𝓝 1) :=
 by { convert tendsto_rpow_div_mul_add (-(1:ℝ)) _ (0:ℝ) zero_ne_one, ring_nf }
 
+/-- The function `exp(x) / x ^ s` tends to `+∞` at `+∞`, for any real number `s` -/
+lemma tendsto_exp_div_rpow_at_top (s : ℝ) : tendsto (λ x : ℝ, exp x / x ^ s ) at_top at_top :=
+begin
+  cases archimedean_iff_nat_lt.1 (real.archimedean) s with n hn,
+  have t := tendsto_exp_div_pow_at_top n,
+  have : 0 < (n:ℝ) - s := by linarith,
+  replace t := tendsto.at_top_mul_at_top t (tendsto_rpow_at_top this),
+
+  have Icieq: eq_on (λ x:ℝ, (exp x / x ^ n) * (x ^ (↑n - s))) (λ x:ℝ, exp x / x ^ s) (Ici 1),
+  { intros x hx,
+    rw [set.Ici, mem_set_of_eq] at hx,
+    have xp : 0 < x := by linarith,
+    ring_nf,
+    rw mul_eq_mul_right_iff, left,
+    rw [sub_eq_neg_add,rpow_add_nat xp.ne', mul_assoc],
+    rw [mul_inv_cancel (pow_ne_zero _ xp.ne'), mul_one],
```

Comment:
You can merge these two lines.

Reply:
@riccardobrasca I have split off this part of the PR into a separate one + applied your comments there. Thanks!